samples/azure-function/README.md

```diff
@@ -0,0 +1,121 @@
+# Use Carbon Aware SDK with an Azure Function
+
+## Overview
+
+The sample included showcase the Azure Functions tooling for the Carbon Aware SDK [C# Class Library](../../docs/architecture/c-sharp-client-library.md). It includes an implementation for `GetAverageCarbonIntensity` and for `GetCurrentForecast`. `GetAverageCarbonIntensity` uses the `EmissionsHandler` to return the carbon intensity rate of a location for a specific timespan. `GetCurrentForecast` uses the `ForecastHandler` to yield the optimal time of a specified location and duration. The functions can run locally for debugging or be deployed to Azure.
```

Comment:
Please add context about WattTime, prerequisites like WattTime user/password and how to configure.

Reply:
Will add link to configuration docs